daemon/daemon_main.go

```diff
@@ -1081,7 +1081,7 @@ func runDaemon() {
 		log.WithError(err).Fatal("Error while creating daemon")
 		return
 	}
-
+	go d.PolicyQueueWorker()
```

Comment:
Since we had to make sure on the order we started this, maybe it is good to put a comment explaining that it has to come before any policyAdd call (e.g. k8s).